hs-convexhull.cabal

```diff
@@ -42,10 +42,9 @@ library
   hs-source-dirs:
       src
   build-depends:
-      Chart ==1.9.3
-    , Chart-cairo ==1.9.3
-    , base >=4.7 && <5
+      base >=4.7 && <5
     , random ==1.1
+    , gloss
```

Comment:
Using stack for this, so this need to update the dependencies in `package.yaml` and maybe `stack.yaml`, as the `.cabal` will get overwritten on builds with stack. 

Just adding `gloss == 1.13.2.1` to the dependencies in `package.yaml` seems to work.